pkg/pbq/store/wal/metrics.go

```diff
@@ -66,13 +66,6 @@ var entryWriteTime = promauto.NewHistogramVec(prometheus.HistogramOpts{
 	Buckets:   prometheus.ExponentialBucketsRange(1, 60, 5),
 }, []string{labelPartitionKey})
 
-var lifespan = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
@yhl25 should we keep the lifespan metric? I think it can be a useful metric when we want to ensure not WAL file lives forever.

Reply:
we can check that using the GC error metric right?

Reply:
if there are errors, that means the file was not deleted

Reply:
There could be cases when GC gets called very late with no errors, causing extremely long lifespan.

Reply:
I think it should be a very rare case. Might not worth monitoring. We can remove lifespan metric for now. +1